src/components/MenuItem/MenuItem.tsx

```diff
@@ -1,28 +1,21 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { MenuItemContent, MenuItemContentProps, MenuItemContentSize } from '@components/MenuItem/MenuItemContent';
+import React, { FC, MouseEvent, useEffect, useState } from 'react';
```

Comment:
Could you please remove `FC` here please. thanks.

Reply:
Sure :)
With the new changes we also need to update our local node version to 18.
I believe that worths an announcement on Slack. WDYT?

Reply:
I plan to make the announcement today.